parm/fixed_files_mapping.yaml

```diff
@@ -1,31 +1,4 @@
 fixed_files:
-  #
```

Comment:
The contents removed here now show up in the `fv3.files_to_link` section of the default config.